example.php

```diff
@@ -4,4 +4,5 @@
 // is_null is a function that does stuff 
 is_null($foo);
 
-?>
\ No newline at end of file
+// Some comment
```

Comment:
```diff
- // is_null is a function that does stuff 
+ // is_null {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester yo

Reply:
@CodelingoBotTester ypo

Reply:
@CodelingoBotTester ypo

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
value of variable rewrite must be quoted

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
unexpected command 

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
value of variable rewrite must be quoted

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
value of variable rewrite must be quoted

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
hi @CodelingoBotTester

Reply:
hi @CodelingoBotTester

Reply:
unexpected command 

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `CodelingoBotTester rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcName "Some comment"

Reply:
@CodeLingoBot rewrite set funcName "Some comment"

Reply:
@CodeLingoBot rewrite set funcComment "Some comment"

Reply:
@CodelingoBotTester rewrite set funcComment "Some comment"

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodeLingoBot to reach us.

</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodeLingoBot to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set funcComment "Some comment"

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>